e2e/execution_test.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/mesg-foundation/engine/execution"
+	"github.com/mesg-foundation/engine/protobuf/acknowledgement"
+	pb "github.com/mesg-foundation/engine/protobuf/api"
+	"github.com/mesg-foundation/engine/protobuf/types"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/grpc/metadata"
+)
+
+func testExecution(t *testing.T) {
+	stream, err := client.ExecutionClient.Stream(context.Background(), &pb.StreamExecutionRequest{
+		Filter: &pb.StreamExecutionRequest_Filter{
+			InstanceHash: testInstanceHash,
+		},
+	})
+	require.NoError(t, err)
+	acknowledgement.WaitForStreamToBeReady(stream)
+
+	ctx := metadata.NewOutgoingContext(context.Background(), passmd)
+	resp, err := client.ExecutionClient.Create(ctx, &pb.CreateExecutionRequest{
+		InstanceHash: testInstanceHash,
+		TaskKey:      "ping",
+		Inputs: &types.Struct{
+			Fields: map[string]*types.Value{
+				"msg": &types.Value{
+					Kind: &types.Value_StringValue{
+						StringValue: "test",
+					},
+				},
+			},
+		},
+	})
+	require.NoError(t, err)
+
+	fmt.Println("recives execution")
```

Comment:
typo `recive` many times

Reply:
Yes, one is suppose to catch `in_progres` status, the second one for `complete`, but becaous service never recived execution, there is no chance to update it.